### DISCUSSION TITLE
How do I add path to laragon environment?

### DISCUSSION THREAD

**Discussion by anchan42** [leokhoa/laragon#522] [Q&A]
I got some program that I want to be abl to run from the terminal.
How could I add the location to the path environment variable?

**Comment by Pen-y-Fan** [comment]
If the program is a stand-alone exe or batch file copy it to the laragon/usr folder. Otherwise, google How to add a folder to `Path` environment variable in Windows

**Reply by alwiismail66** [reply]
i think how to the other task in cmder like bash etc read the laragon path environment so when i use other task in cmder can automatic detect the path from laragon not user path environment
